### ISSUE TITLE
Need All menus of request and response in single Line

### ISSUE THREAD

**Issue by MuraliKrishna4550** [a85/POSTMan-Chrome-Extension#8806]
**Describe the bug**
Need All menus of request and response in single Line
The previous version will be like that, all menus of req and response are in a single line if I scroll request panel too small all menus in response visible in a single line, same for request also.

**Expected behavior**
All menus should be in single line.

**Screenshots**
If applicable, add screenshots to help explain your problem. Please make sure the screenshot does not contain any sensitive information such as API keys or access tokens. 

![Capture](https://user-images.githubusercontent.com/49240755/88018190-d03b8e00-cb44-11ea-9e98-0dd00c764f60.PNG)
**App information (please complete the following information):**
 - App Type [e.g. Chrome App, Native App]
 - Postman Version [e.g. 6.0.1]
 - OS: [e.g. macOS High Sierra 10.13.2]

**Additional context**

**Comment by coditva** [comment]
Hey @MuraliKrishna4550,

The tabs change to dropdown when there isn't enough space in the response panel to show the tabs. You can change the dropdown to tabs by simply resizing the response pane to a bigger size.

![collapse-tabs](https://user-images.githubusercontent.com/8445768/88018929-4b517400-cb46-11ea-8962-607f3fd77689.gif)

If you don't have enough space inside the Postman window, you can switch to a single-pane layout for wider space.

![image](https://user-images.githubusercontent.com/8445768/88018973-62906180-cb46-11ea-8b7d-d4588a6a3009.png)


Does this solve your use-case? If not, can you elaborate on that?

**Comment by MuraliKrishna4550** [comment]
I need a two-pane view and I don't want to scroll like that, the previous version it used to work fine. Please send me a link to download the previous version I will use that one, and will not upgrade

**Comment by MuraliKrishna4550** [comment]
I want like this, how it works as previous versions. please send me
previous version link.

**Comment by jerrfi** [comment]
I agree, this "improvement" is much more like a bug and a big step backward in usability for the Two Pane View.  And the Single Pane View is very limiting since it forces too much back & forth scrolling in 2 windows to synch their contents.

Please return to the fixed tabs for both panes.  

Alternately, at least include the Test Results success failure ratio when the tabs go to a drop down.   Examples: (64/64) or (57/64)

Thanks
